Firestore/core/src/firebase/firestore/util/CMakeLists.txt

```diff
@@ -17,3 +17,16 @@ add_library(
   autoid.cc
   secure_random_arc4random.cc
 )
+
+# Apple-specific utility build.
+# Expose FirebaseCore/FIRLogger.h to cmake.
+# add_library(
+#   firebase_firestore_util_apple
+#   log_apple.mm
+# )
+
+# Non-Apple-specific utility build.
+add_library(
```

Comment:
For now, just make log_other a member of the firebase_firestore_util library. Let's add the conditional build logic separately.